pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -657,3 +655,21 @@ func getNodeMAC(node *corev1.Node) (net.HardwareAddr, error) {
 	}
 	return mac, nil
 }
+
+func (c *Controller) getNodeTransportAddress(node *corev1.Node) (net.IP, error) {
+	if c.networkConfig.TransportIface != "" {
+		transportAddrStr := node.Annotations[types.NodeTransportAddressAnnotationKey]
+		if transportAddrStr != "" {
+			peerNodeAddr := net.ParseIP(transportAddrStr)
+			return peerNodeAddr, nil
```

Comment:
It should check if it's valid when it's set. Otherwise the nil value may affect the following process, even crash the process.

Reply:
Got it, I will add check.